tools/offsets-tool-py/offsets-tool.py

```diff
@@ -84,6 +86,14 @@ def require_emscipten_path (args):
 			self.target = Target ("TARGET_WASM", None, [])
 			self.target_args += ["-target", args.abi]
 
+		# Linux
+		elif "arm-linux-gnueabihf" == args.abi:
+			self.target = Target ("TARGET_ARM", None, ["ARM_FPU_VFP", "HAVE_ARMV5", "HAVE_ARMV6", "HAVE_ARMV7"] + LINUX_DEFINES)
+			self.target_args += ["--target=arm---gnueabihf"]
+			self.target_args += ["-I", "/usr/lib/gcc-cross/arm-linux-gnueabihf/8/include/"]
+			self.target_args += ["-I", "/usr/lib/gcc-cross/arm-linux-gnueabihf/8/include-fixed/"]
```

Comment:
Maybe `arm-linu-gnuabihf` on these lines, change to `args.abi`?

Reply:
can we make these configurable too? on my Ubuntu 18.04 I only get `/usr/lib/gcc-cross/arm-linux-gnueabihf/7`

Reply:
Check if the directories exist?
If none at all, error?

Reply:
Try to generalize for any target? 
Gcc doesn’t do some of this itself?
Like, mainly just run the correct gcc?

Reply:
@jaykrell this uses `clang`. Thanks for the comments, updated the PR.

Reply:
Sorry - clang doesn’t pick up these
Include directories itself? I’ll try later..